include/cute/atom/copy_traits_xe.hpp

```diff
@@ -408,25 +408,25 @@ CUTE_HOST_DEVICE constexpr auto make_fragment_layout(TiledCopy &tiled_copy,
   auto [mma_atom_shape, total_mma_atom_iters_M, total_mma_atom_iters_N] = fragment_top_level_shape;
   auto mma_atom_shape_2d = prepend<2>(mma_atom_shape, _1{});
 
-  Int mma_atom_size_M =
+  auto mma_atom_size_M =
       Int<!TiledCopy::is_convention_MN ? size<0>(mma_atom_shape_2d) : size<1>(mma_atom_shape_2d)>{};
```

Comment:
I think it is more readable to write:
```suggestion
  constexpr int mma_atom_size_M =
      !TiledCopy::is_convention_MN ? size<0>(mma_atom_shape_2d) : size<1>(mma_atom_shape_2d);
```
Of course if all these get changed from `Int` to `int` means that later things need to be converted to `Int`. E.g. 
```
make_shape(mma_atom_iters_in_copy_M, copy_iters_M)
```
would need to change to 
```
make_shape(Int<mma_atom_iters_in_copy_M>{}, Int<copy_iters_M>{})
```
or 
```
Shape<Int<mma_atom_iters_in_copy_M>, Int<copy_iters_M>>{}
```